backend/de.metas.business/src/main/java/de/metas/freighcost/FreightCostRepository.java

```diff
@@ -209,11 +208,13 @@ private static FreightCostShipper toFreightCostShipper(
 	private FreightCostBreak toFreightCostBreak(final I_M_FreightCostDetail record)
 	{
 		final CountryId countryId = CountryId.ofRepoIdOrNull(record.getC_Country_ID());
+		final CountryAreaId countryAreaId = CountryAreaId.ofRepoIdOrNull(record.getC_CountryArea_ID());
```

Comment:
AFAIU users could currently still have multiple breaks for the same shipmentValue and shipper and idk what kinds of problems that will cause down the road.
therefore i suggest to add a SeqNo (default value expression with a "max(...)+10"-SQL ) with a unique constraint and take the matching break with the smallest shipmentValue **and seqNo**

Reply:
added `M_FreightCostShipper.SeqNo` and changed sort order to use it (records with a null SeqNo will have least priority).